httpclient/src/test/java/feign/httpclient/ApacheHttpClientTest.java

```diff
@@ -188,6 +188,20 @@ public void postWithSpacesInPath() throws IOException, InterruptedException {
                 .hasBody("foo");
     }
 
+    @Test
```

Comment:
added https://github.com/Netflix/feign/commit/b29f9d8aea5a326069275d9787eebe26a09930e3 to hopefully explain the intent of client tests